index.html

```diff
@@ -8,27 +8,27 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <div class="container" style="background-image:url('/home/ignatius/Documents/Microverse Projects/Portfolio/images/background-image.png');">
-        <div class="toolbar">
-            <p class="heading-1" style="color:blue ;">My logo</p>
+    <header class="container">
+        <nav class="toolbar">
+            <a href="#">My logo</a>
+            <img src="images/hamburger.png" alt="hamburger">
+        </nav>
+        <section class="greetings">
+            <p>I am Ignatius Cheruiyot. Glad to meet you!</p>
+        </section>
+        <section class="about-me">
+            <p>I am a software developer and I can help you build a project,feature or website. If you will like what you see and have a project you will need coded, don't hesitate to contact me, look through some of my work and experince.</p> 
+        </section>
+        <div class="heading">
+            </p>LET'S CONNECT</p> 
         </div>
-        <div>
-            <h2 class="greetings">I am Ignatius Cheruiyot</h2>
-            <h2 class="greetings">Glad to meet you!</h2>
-            <div class="about-me">
-                <p>I am a software developer and I can help you build a project,feature or website.</p>
-                <p>If you will like what you see and have a project you will need coded,don't hesitate to contact me</p> 
-                <p>look through some of my work and experince</p>
-            </div>
-            <h3 class="heading" style="color:blue;">LET'S CONNECT</h3>
-            <div class="icon-bar">
-                <img src="/home/ignatius/Documents/Microverse Projects/Portfolio/images/twitter.png" alt="">
-                <img src="/home/ignatius/Documents/Microverse Projects/Portfolio/images/linkedin.png" alt="">
-                <img src="/home/ignatius/Documents/Microverse Projects/Portfolio/images/m.png" alt="">
-                <img src="/home/ignatius/Documents/Microverse Projects/Portfolio/images/hand.png" alt="">
-                <img src="/home/ignatius/Documents/Microverse Projects/Portfolio/images/cat.png" alt="">
-            </div>
-        </div>
-     </div>
+        <section class="icon-bar">
+            <li><img src="images/twitter.png" alt="twitter"></li>
+            <li><img src="images/linkedin.png" alt="linkedin"><li>
+            <li><img src="images/m.png" alt="m"></li>
+            <li><img src="images/hand.png" alt="hand"></li>
+            <li><img src="images/cat.png" alt="cat"></li>
+        </section>     
+    </header>
```

Comment:
Please consider removing everything else from the header tag except the `logo and hamburger icon` the rest of the content needs to be wrapped in a single `section` element. There is no need to wrap everything in a section tag as you did. Its is considered to be good for semantics.